libs/dh/metering-point/feature-overview/src/components/measurement/dh-measurements.component_v2.ts

```diff
@@ -0,0 +1,154 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { formatNumber } from '@angular/common';
+import { Component, computed, effect, inject, input, LOCALE_ID } from '@angular/core';
+import { TranslocoDirective } from '@jsverse/transloco';
+
+import { dayjs, WattSupportedLocales } from '@energinet-datahub/watt/date';
+import { VaterUtilityDirective } from '@energinet-datahub/watt/vater';
+import { WattDataFiltersComponent, WattDataTableComponent } from '@energinet-datahub/watt/data';
+import { WATT_TABLE, WattTableColumnDef, WattTableDataSource } from '@energinet-datahub/watt/table';
+
+import { lazyQuery } from '@energinet-datahub/dh/shared/util-apollo';
+import {
+  GetMeasurementsById_V2Document,
+  Resolution,
+} from '@energinet-datahub/dh/shared/domain/graphql';
+
+import { MeasurementPositionV2, QueryVariablesV2 } from '../../types';
+import { DhMeasurementsFilterComponent } from './dh-measurements-filter.component';
+
+@Component({
+  selector: 'dh-meter-data-v2',
+  imports: [
+    TranslocoDirective,
+    WATT_TABLE,
+    WattDataTableComponent,
+    WattDataFiltersComponent,
+    VaterUtilityDirective,
+    DhMeasurementsFilterComponent,
+  ],
+  template: `
+    <watt-data-table
+      vater
+      inset="ml"
+      [enableSearch]="false"
+      [enableCount]="false"
+      [error]="query.error()"
+      [ready]="query.called()"
+      [enablePaginator]="false"
+      *transloco="let t; read: 'meteringPoint.measurements'"
+    >
+      <watt-data-filters>
+        <dh-measurements-filter (filter)="fetch($event)" />
+      </watt-data-filters>
+      <watt-table
+        *transloco="let resolveHeader; read: 'meteringPoint.measurements.columns'"
+        [resolveHeader]="resolveHeader"
+        [columns]="columns()"
+        [dataSource]="dataSource"
+        [loading]="query.loading()"
+        sortDirection="desc"
+        [sortClear]="false"
+      >
+        <ng-container *wattTableCell="columns().observationTime; let element">
+          {{ this.formatObservationTime(element.observationTime, element.current.resolution) }}
```

Comment:
suggestion: we should consider creating a pipe for this because the function is called quite a lot - on initial load as well as on every change detection cycle. Can be a separate PR

Reply:
Good idea, will have a look at it in the next PR 👍🏻